pkg/controller/cloneset/core/cloneset_core.go

```diff
@@ -95,7 +95,7 @@ func (c *commonControl) newVersionedPods(cs *appsv1alpha1.CloneSet, revision str
 		}
 		clonesetutils.WriteRevisionHash(pod, revision)
 
-		pod.Name = fmt.Sprintf("%s-%s", cs.Name, id)
+		pod.Name = fmt.Sprintf("%s-%s", cs.GenerateName, id)
```

Comment:
shall we leave pod name empty and let the apiserver assign unique names using generateName?